src/event/v03/attributes.rs

```diff
@@ -42,9 +42,10 @@ impl<'a> IntoIterator for &'a Attributes {
     }
 }
 
+#[derive(PartialEq, Debug, Clone, Copy)]
 pub struct AttributesIntoIterator<'a> {
```

Comment:
Same, Does it work with `pub(crate)`?

Reply:
no because Attributes itself is `pub`. However I have changed the mod files to limit it to being `pub(crate)`

Reply:
I don't get why the `Attributes` visibility is a problem connected to this iterator struct

Reply:
The `IntoIterator` trait implemented for `Attributes` uses the `AttributesIntoIterator` struct. Since `Attributes` is public, that would cause a leak of the crate-visible type.

Reply:
ok let me try locally